diff --git a/dist/index.d.ts b/dist/index.d.ts
index d680a54d725d462ee3141bcf913d8d5bb5fc0912..917c960d9cebc898372ddb3d0a95073fa14d9c7e 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -102,8 +102,8 @@ type EventHandlersMap<EventMap> = {
  *   touchend: touchHandler
  * });
  */
-declare function createEventListenerMap<EventMap extends Record<string, Event>, UsedEvents extends keyof EventMap = keyof EventMap>(target: MaybeAccessor<Many<EventTarget>>, handlersMap: Partial<Pick<EventHandlersMap<EventMap>, UsedEvents>>, options?: EventListenerOptions): void;
 declare function createEventListenerMap<Target extends TargetWithEventMap, EventMap extends EventMapOf<Target>, HandlersMap extends Partial<EventHandlersMap<EventMap>>>(target: MaybeAccessor<Many<Target>>, handlersMap: HandlersMap, options?: EventListenerOptions): void;
+declare function createEventListenerMap<EventMap extends Record<string, Event>, UsedEvents extends keyof EventMap = keyof EventMap>(target: MaybeAccessor<Many<EventTarget>>, handlersMap: Partial<Pick<EventHandlersMap<EventMap>, UsedEvents>>, options?: EventListenerOptions): void;
 
 type WindowEventProps = {
     [K in keyof WindowEventMap as `on${Capitalize<K>}` | `on${K}`]?: (event: WindowEventMap[K]) => void;